xain_fl/config/schema.py

```diff
@@ -268,10 +273,11 @@ def create_class_from_schema(class_name: str, schema: Schema) -> Any:
         A new class where attributes are the given schema's keys
     """
     # pylint: disable=protected-access
-    keys = schema._schema.keys()
-    attributes = list(
-        map(lambda key: key._schema if isinstance(key, Schema) else key, keys)
-    )
+    attributes = []
+    for key in schema._schema.keys():
+        if isinstance(key, Schema):
+            key = key._schema
```

Comment:
Ah this is because I experimented with a new configuration section while working on this PR, and I needed to make this function a bit smarter. I ended up giving up and reverted my changes, but this slipped through the cracks. I'll undo it.

Reply:
reverted the changes